python/cuda_cccl/tests/parallel/examples/scan/basic_scan.py

```diff
@@ -32,49 +31,44 @@ def add_op(a, b):
     return d_output.get()
 
 
-def exclusive_scan_example():
-    """Demonstrate exclusive scan with max operation."""
-
-    def max_op(a, b):
-        return max(a, b)
+def inclusive_scan_custom_op():
+    """Inclusive scan with custom operation (prefix sum of even values)."""
 
-    h_init = np.array([1], dtype="int32")
-    d_input = cp.array([-5, 0, 2, -3, 2, 4, 0, -1, 2, 8], dtype="int32")
+    h_init = np.array([0], dtype="int32")
+    d_input = cp.array([1, 2, 3, 4, 5], dtype="int32")
     d_output = cp.empty_like(d_input, dtype="int32")
 
-    # Run exclusive scan with automatic temp storage allocation
-    parallel.exclusive_scan(d_input, d_output, max_op, h_init, d_input.size)
+    def add_op(a, b):
+        return (a if a % 2 == 0 else 0) + (b if b % 2 == 0 else 0)
 
-    # Check the result is correct
-    expected = np.asarray([1, 1, 1, 2, 2, 2, 4, 4, 4, 4])
+    parallel.inclusive_scan(d_input, d_output, add_op, h_init, d_input.size)
+
+    expected = np.asarray([0, 2, 2, 6, 6])
     assert np.array_equal(d_output.get(), expected)
-    print(f"Exclusive scan result: {d_output.get()}")
+    print(f"Inclusive scan result: {d_output.get()}")
     return d_output.get()
 
 
-def prefix_sum_example():
-    """Simple prefix sum example using inclusive scan."""
-
-    def add_op(a, b):
-        return a + b
+def exclusive_scan_example():
+    """Exclusive scan (prefix sum)."""
 
     h_init = np.array([0], dtype="int32")
-    d_input = cp.array([1, 2, 3, 4, 5], dtype="int32")
+    d_input = cp.array([-5, 0, 2, -3, 2, 4, 0, -1, 2, 8], dtype="int32")
     d_output = cp.empty_like(d_input, dtype="int32")
 
-    # Run inclusive scan with automatic temp storage allocation
-    parallel.inclusive_scan(d_input, d_output, add_op, h_init, d_input.size)
+    parallel.exclusive_scan(
+        d_input, d_output, parallel.OpKind.PLUS, h_init, d_input.size
```

Comment:
Lets keep one example with MAX instead of PLUS

Reply:
Max doesn't work yet (github.com/NVIDIA/cccl/issues/5515)